### ISSUE TITLE
REST Power On  is failing, 'REST_USERNAME' is not defined

### ISSUE THREAD

**Issue by ssombar** [openbmc/openbmc-test-automation#1540]
REST Power On  stack_mode=skip   is failing

```
Traceback (most recent call last):

   OPENBMC_HOST, REST_USERNAME,

NameError: global name 'REST_USERNAME' is not defined
```

**Comment by ssombar** [comment]
george says  "lib/resource.txt:${REST_PASSWORD}       0penBmc"
and I have  -v REST_USERNAME:root -v REST_PASSWORD:0penBmc  in my robot call
I'll dig a bit more on this to see if I can narrow the failure down a bit

**Comment by ssombar** [comment]
in "obmc_boot_test.py" it looks to me that control is never returned back to routine obmc_boot_test_py(loc_boot_stack=None
once setup() is called at line 1051

Test case:
```Suite Setup Execution
    [Documentation]  Do the pre-test setup.
    REST Power On  stack_mode=skip```

**Comment by ssombar** [comment]
It appears that there may already be a fix for this, gerrit #17818.

**Comment by gkeishin** [comment]
Closing this then,
